dolphin/drivers/dellemc_storage/vmax/vmax.py

```diff
@@ -90,7 +90,8 @@ def get_storage(self, context):
         return storage
 
     def list_pools(self, context):
-        pass
+        self._check_connection(context)
```

Comment:
it seems check connection is not required.

Reply:
Removed